util.py

```diff
@@ -260,14 +260,22 @@ def create_executable(filename, command):
     )
 
 
-def check_file_sha256_checksum(filename, reference_checksum):
+def compute_file_sha256_checksum(filename):
```

Comment:
Many libraries, such as numpy, tensorflow and many others, have a "tests" package to allow their users to test their code using the library itself. Maybe doing something like that can be more explicit than putting this code into a "util.py" file

Reply:
This function is not only used in testing, but also at other jobs (e.g. the "Download" job). So this is why it was in util.py already.